src/game/common/system/mempoolobj.h

```diff
@@ -76,45 +76,44 @@ private: \
     virtual MemoryPool *Get_Object_Pool() override { return Get_Class_Pool(); } \
 \
 public: \
-    enum classname##MagicEnum{ classname##_GLUE_NOT_IMPLEMENTED }; \
+    enum classname##MagicEnum{ classname##_GLUE_NOT_IMPLEMENTED = 0 }; \
 \
-    void *operator new(size_t size); \
-    void *operator new(size_t size, classname##MagicEnum) \
+    void *operator new(size_t size, classname##MagicEnum = classname##_GLUE_NOT_IMPLEMENTED) \
```

Comment:
`MagicEnum` argument now has default value here. This allows writing `new MyPoolObject`.